docs/zh-CN/docs/sql-manual/sql-functions/json-functions/jsonb_valid.md

```diff
@@ -0,0 +1,75 @@
+---
+{
+    "title": "jsonb_valid",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## jsonb_valid
+### description
+
+jsonb_valid 函数返回0或1以表明是否为有效的JSON, 如果参数是NULL则返回NULL。
```

Comment:
What's the difference between jsonb_valid and json_valid? since we already have a function json_valid.

Reply:
This pr from this https://github.com/apache/doris/issues/21268.  Some JSONB functions which are not implement but JSON has already implement.The func jsonb_valid is one of them.